instrumentation/src/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xmxl version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="org.oppia.android.app.instrumentation">
+  <uses-sdk
+    android:minSdkVersion="19"
+    android:targetSdkVersion="28" />
```

Comment:
Should this be 29? That's what the test binary is set to.

Reply:
Done